middleware-catalog-module/address/Address/src/main/java/co/tpg/catalog/function/AddressFunction.java

```diff
@@ -0,0 +1,156 @@
+package co.tpg.catalog.function;
+
+import co.tpg.catalog.dao.DAO;
+import co.tpg.catalog.dao.exception.BackendException;
+import co.tpg.catalog.exception.ProcessingException;
+import co.tpg.catalog.model.Address;
+import co.tpg.catalog.request.HttpMethod;
+import co.tpg.catalog.request.AddressRequest;
+import co.tpg.catalog.response.AbstractResponse;
+import co.tpg.catalog.response.ErrorResponse;
+import co.tpg.catalog.response.AddressListResponse;
+import co.tpg.catalog.response.AddressResponse;
+import co.tpg.catalog.dao.AddressDAO;
+import com.amazonaws.AmazonServiceException;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBMapper;
+import com.amazonaws.services.dynamodbv2.model.ResourceNotFoundException;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+
+
+/**
+ * Handler for requests to Address.
+ * @author maxx
+ * @since 2019-10-11
+ */
+public class AddressFunction implements RequestHandler<AddressRequest, AbstractResponse> {
+
+    public AbstractResponse handleRequest(final AddressRequest input, final Context context) {
+        final String DYNAMO_TABLE_NAME = "Address";
+        final AmazonDynamoDB dynamoDB = AmazonDynamoDBClientBuilder.defaultClient();
```

Comment:
After using DAO the AmazonDynamoDB must disappear from here. It means something is wrong.